config/locales/two_factor_authentication/fr.yml

```
@@ -1,6 +1,12 @@
 ---
 fr:
   two_factor_authentication:
+    aal2_request:
+      phishing_resistant_html: '%{sp_name} nécessite une méthode d’authentification de
+        haute sécurité, telle que le déverrouillage du visage ou du tactile, une
+        clé de sécurité ou un identifiant d’employé du gouvernement.'
+      piv_cac_only_html: '%{sp_name} nécessite votre identifiant d’employé du
```

ditto for strong tags
```suggestion
      phishing_resistant_html: '<strong>%{sp_name}</strong> nécessite une méthode d’authentification de
        haute sécurité, telle que le déverrouillage du visage ou du tactile, une
        clé de sécurité ou un identifiant d’employé du gouvernement.'
      piv_cac_only_html: '<strong>%{sp_name}</strong> nécessite votre identifiant d’employé du
```

---

Fixed missing tags in https://github.com/18F/identity-idp/commit/f00646043a5ae487d65b613a417fa6280562d46e.